### PR TITLE
Imporving an error message(#13334)

### PR DESCRIPTION
**Pull request by rafiwts**
Resolving an issue (#13334) of code with the vague error description.

### CODE REVIEW

File: saleor/core/exceptions.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def __init__(self, message=None, *, permissions: Optional[Iterable[Enum]] = None
             if permissions:
                 permission_list = ", ".join(p.name for p in permissions)
                 message = (
-                    f"You need one of the following permissions: {permission_list}"
+                    f"To access this path, you need following permissions: {permission_list}"
```
Comment (maarcingebala) [inline saleor/core/exceptions.py:66]: Grammar fix:
```suggestion
                    f"To access this path, you need the following permissions: {permission_list}"
```
Not sure if it doesn't exceed the line limit after this change, so the string would need to be split into two lines.

Comment (rafiwts) [reply saleor/core/exceptions.py]: All aforementioned remarks have been taken into account and corrections have been implemented.

**Review by maarcingebala — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

**Comment by maarcingebala** [comment]
@rafiwts Hey, could you rebase your PR and move the changelog entry to 3.16 section? Scope for 3.15 is now closed and any new changes will be released as 3.16.

**Comment by rafiwts** [comment]
@maarcingebala Instead of rebasing, I have added a new commit with all necessary changes. Hopefully, it does not have any influence on the PR.

**Comment by maarcingebala** [comment]
@rafiwts Thank you for contributing to Saleor 🎉 !

**Pull request merged**